### PR TITLE
Login with Social Media

### LINKED ISSUE #463: EVM Wallet - Add Telos Cloud Wallet (OreId) using WalletConnect.
## Overview
Add "Telos Cloud Wallet" (OreId Auth) using WalletConnect.

Currently, the OreID is separated from the rest of connecting methods, including WalletConnect. This issue requires adding the OreId option to the WalletConnect list.

Question: Since the OreId button works on mobile by itself, do we need this solution?

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #463

## Description
This PR includes a new button for the Home page (on the EVM side) which provides the users a way to authenticate using social media providers without even having a wallet installed.

## Notes
- I removed the timeout when sending tokens because it was annoying. Maybe we should discuss if we want this timeout or not.
- I'm currently using [my own app id for testing because I needed to do some tests directly](https://github.com/telosnetwork/telos-wallet/issues/492#issuecomment-1646256233). We need to decide which Identity providers are we going to be using and enable them in the app registration.

## Test scenarios
- go to [this link](https://deploy-preview-490--wallet-staging.netlify.app/)
- Press the "Login with Social Media" button
   - A menu with 4 social media authentication providers should appear
- Choose any provider and press the button
   - a window will pop up to ask you for a pin (If is your first time, it will ask you to repeat the pin)
   - after confirmation
     - the window will load and close automatically
     - you will be logged in with a zero balance account
- Copy your address, and send to that address some tokens from another account you control
- After you have some tokens in your balance, go to send page and try to send them (make sure you have some TLOS)
- When pressing Send button, a window will pop up
- Enter PIN and confirm
  - the transaction should go with no problems.

<table>
<tr>
<td></td>
<td></td>
</tr>
</table>

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness
-   [x] I have added appropriate test coverage

### CODE REVIEW

**Review by ezra-guzman — APPROVED** [review]
works very well for me on desktop and mobile, awesome job 💯

**Review by donnyquixotic — CHANGES_REQUESTED** [review]
EDIT: this is for mainnet, testnet is working as expected. We should update the official app ids for both as part of this PR

I'm receiving a zero balance error preventing send transaction (account has available TLOS) using google via oreid:
image

**Review by donnyquixotic — APPROVED** [review]

**Comment by donnyquixotic** [comment]
- [ ] update appId (testnet)
- [ ] update appId (mainnet)

**Comment by Viterbo** [comment]
Re-opening this PR that I think was closed by mistake

**Review by donnyquixotic — APPROVED** [review]

File: src/antelope/wallets/authenticators/OreIdAuth.ts [hunk 1]
```diff
@@ -233,7 +233,7 @@ export class OreIdAuth extends EVMAuthenticator {
         const transaction = await oreId.createTransaction({
             transaction: transactionBody,
             chainAccount: from,
-            chainNetwork: ChainNetwork.TelosEvmTest,
+            chainNetwork: this.getChainNetwork(useChainStore().getChain(this.label).settings.getNetwork()),
```
Comment (Viterbo) [inline src/antelope/wallets/authenticators/OreIdAuth.ts:236]: Nice !!! :+1:

**Pull request merged**